editor/components/inserter/group.js

```diff
@@ -39,6 +39,14 @@ export default class InserterGroup extends Component {
 		}
 	}
 
+	hoverBlock( block ) {
```

Comment:
The function name doesn't seem entirely accurate. This is not the hover handler itself; it creates a hover handler. Maybe: `createToggleBlockHover` ?

Also: Maybe it's worthwhile to check `this.props.onHover` before returning anything here? That way, we could e.g. `return null;` and avoid creating new function references (i.e. forcing React to reconcile changing function handlers).